Pod/Classes/NSManagedObject+VOKManagedObjectAdditions.h

```diff
@@ -97,6 +97,30 @@
 + (NSArray *)vok_fetchAllForPredicate:(NSPredicate *)predicate forManagedObjectContext:(NSManagedObjectContext *)contextOrNil;
 
 /*
+ Returns all entites matching the predicate.
+ @param predicate       Predicate to use to fetch.
+ @param sortDescriptors SortDescriptors to use to sort the results.
+ @param contextOrNil    The managed object context to fetch in.  If nil, the main context will be used.
+ @return                NSArray full of the instances of the current class.
+ */
++ (NSArray *)vok_fetchAllForPredicate:(NSPredicate *)predicate
+                             sortedBy:(NSArray *)sortDescriptors
+              forManagedObjectContext:(NSManagedObjectContext *)contextOrNil;
+
+/*
+ Returns all entites matching the predicate.
```

Comment:
this docu-comment should also be updated to mention the sorting + ascending option